packages/app/hooks/select/useSelect.ts

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
```

Comment:
this hook might be leftover logic from the docs or something, i don't think we need